### PR TITLE
Requested Features - Personalisation

### PR DESCRIPTION
**Pull request by manstheram**
This pull-request includes:
- smaller lock flyouts in general (less space is wasted)
- vertical volume bars
- better inactive controls
- fixed start-up task
- and icon only lock flyouts.
This is finished. @Samuel12321

### CODE REVIEW

**Comment by Samuel12321** [comment]
Thanks @manstheram , will review test and merge this evening.

**Comment by manstheram** [comment]
I just noticed that the vertical volume bar setting doesn't save when you reopen the app, but the consise lock flyouts one does, so I gotta fix that.

**Review by Samuel12321 — CHANGES_REQUESTED** [review]
Hi @manstheram , Thanks for the PR.
Just found a few issues that need to be resolved before merging. 
- Flyout axis setting is currently reversed
- Lock icons are not following flyout theme, so are barely visible when flyout theme is dark.
- Lock icons really need to be Segoe MDL2 or Segoe UI for consistency within both ModernFlyouts and Windows its self

File: ModernFlyouts/Properties/Strings.resx [hunk 1]
```diff
@@ -497,4 +497,10 @@
   <data name="Settings.VerticalAlignment" xml:space="preserve">
     <value>Vertical alignment</value>
   </data>
+  <data name="Settings.FlyoutAxis" xml:space="preserve">
+    <value>Change axis of flyout</value>
```
Comment (Samuel12321) [inline ModernFlyouts/Properties/Strings.resx:501]: "Flyout Axis:" as the header may be more concise

File: ModernFlyouts/Properties/Strings.resx [hunk 2]
```diff
@@ -497,4 +497,10 @@
...
+  </data>
+  <data name="Settings.UseSmallFlyout" xml:space="preserve">
+    <value>Use consise flyouts for the lock flyouts</value>
```
Comment (Samuel12321) [inline ModernFlyouts/Properties/Strings.resx:504]: "Enable Condensed Lock Flyouts:"

File: ModernFlyouts/Properties/Strings.resx (hunk 1 above)
Comment (manstheram) [reply ModernFlyouts/Properties/Strings.resx]: Changed

File: ModernFlyouts/Properties/Strings.resx (hunk 2 above)
Comment (manstheram) [reply ModernFlyouts/Properties/Strings.resx]: Changed

**Comment by manstheram** [comment]
> Lock icons really need to be Segoe MDL2 or Segoe UI for consistency within both ModernFlyouts and Windows its self

About that, there are no lock icons for each type of lock in those fonts, which is why I had to use custom ones. ShankarBUS tried to do a similar thing, however he could only get blurry images since he was not using svgs - #63 

> But I can't create a neat icon. They looked blurry when scaled. WPF doesn't support SVGs. There were too many issues with this.
I guess just showing the lock icon will be enough for now

I got around the blurry state by using a converter "SvgtoXaml" to make svgs into shapes that I can use in WPF. [https://github.com/BerndK/SvgToXaml](url)

**Comment by manstheram** [comment]
I have now added those first two changes you needed, but the third one can't be done cause there are no icons to use from those fonts. It should be finished now.

**Review by manstheram — COMMENTED** [review]
It should be ready to use.

**Comment by werker03** [comment]
Request Feature: Manual Audio Output in ModernFlyouts

I got a Soundmixer program running so my default audio mismatches with my headset. Windows 10 showed me my Headset tho when using the audio wheel at the back of my headset, which is good! I just want to know what audio lvl my headset is on.

Can we get an option for ModernFlyouts to set the audio displayed in it to set it manually which is shown when using the flyout?

**Comment by werker03** [comment]
> This pull-request includes:
> 
> * smaller lock flyouts in general (less space is wasted)
> * vertical volume bars
> * better inactive controls
> * fixed start-up task
> * and icon only lock flyouts.
>   This is finished. @Samuel12321

Request Feature: Manual Audio Output in ModernFlyouts

I got a Soundmixer program running so my default audio mismatches with my headset. Windows 10 showed me my Headset tho when using the audio wheel at the back of my headset, which is good! I just want to know what audio lvl my headset is on.

Can we get an option for ModernFlyouts to set the audio displayed in it to set it manually which is shown when using the flyout?

**Comment by Samuel12321** [comment]
> Request Feature: Manual Audio Output in ModernFlyouts
> 
> I got a Soundmixer program running so my default audio mismatches with my headset. Windows 10 showed me my Headset tho when using the audio wheel at the back of my headset, which is good! I just want to know what audio lvl my headset is on.
> 
> Can we get an option for ModernFlyouts to set the audio displayed in it to set it manually which is shown when using the flyout?

@werker03 this is not the place to request features.
Please file a feature request in the issues tab.

**Pull request merged**